packages/console/src/pages/EnterpriseSsoDetails/Connection/XmlFileReader/index.tsx

```diff
@@ -24,12 +26,8 @@ type Props = {
 };
 
 function XmlFileReader({ onChange, value }: Props) {
-  /**
-   * It's ok to use 0 as a fallback file size because it will not be displayed if the value is empty.
-   */
-  const [fileSize, setFileSize] = useState<number>(
-    conditional(value && getXmlFileSize(value)) ?? 0
-  );
+  const theme = useTheme();
+  const fileSize = useMemo(() => getXmlFileSize(value ?? ''), [value]);
```

Comment:
Why the `acceptedFile.size` does not work?

Reply:
If the content does not come from file (load from DB), then `acceptedFile.size` should not work?

Reply:
Need to add this info in the PR summary.

Reply:
I don't think we need the `useMemo` here since `getXmlFileSize` is an external util method

Reply:
Calculated the file size in-place and removed the `useMemo()` and also added the reason why `acceptedFile.size` method is not applicable in our case to PR description.